pkgs/applications/display-managers/greetd/qtgreet/default.nix

```
@@ -0,0 +1,58 @@
+{
+  stdenv,
+  lib,
+
+  fetchFromGitLab,
+
+  wrapQtAppsHook,
+  meson,
+  pkg-config,
+  cmake,
+  qttools,
+  ninja,
+
+  qtbase,
+  json_c,
+  wlroots,
+  wayqt,
+  wayland,
+  pixman
+}:
+
+stdenv.mkDerivation (finalAtrrs: {
```

```suggestion
stdenv.mkDerivation {
```
Now this is fully unused.